datalad/cmd.py

```diff
@@ -433,19 +434,50 @@ class GitRunner(Runner):
     GIT_WORK_TREE environment variables set to the absolute path
     if is defined and is relative path
     """
+    _GIT_PATH = None
+
+    @borrowdoc(Runner)
+    def __init__(self, *args, **kwargs):
+        super(GitRunner, self).__init__(*args, **kwargs)
+        self._check_git_path()
+
+    @staticmethod
+    def _check_git_path():
+        """If using bundled git-annex, we would like to use bundled with it git
+
+        Thus we will store _GIT_PATH a path to git in the same directory as annex
+        if found.  If it is empty (but not None), we do nothing
+        """
+        if GitRunner._GIT_PATH is None:
+            out, err = Runner().run('which git-annex')  # figure out which git-annex
```

Comment:
oh right... but then what to do in except?  if on windows -- we can just assume bundled (thus not even get into try/except).  but if it blows and not on windows -- use non-bundled (default) with a warning?